whois-endtoend/src/test/java/net/ripe/db/whois/query/InverseQueryTestIntegration.java

```diff
@@ -266,7 +274,219 @@ public void inverse_email_from_untrusted_then_fail() {
         assertThat(response, containsString("is not an inverse searchable attribute"));
     }
 
+    @Test
+    public void inverse_email_charset() {
```

Comment:
Move charset query tests into their own class (i.e. they are unrelated to inverse queries)